doc/lessons/05_fonts_utf8.md

```
@@ -0,0 +1,235 @@
+pdf.SetFontLocation("font/th")  // Point to font directory
+pdf.AddPage()
+
+// Auto-loads from font/th/Kanit/ subfolder
+pdf.SetFont("Kanit", "", 14)
+pdf.Cell(0, 10, "Auto-loaded from filesystem")
+
+_ = pdf.OutputFileAndClose("external.pdf")
+```
+
+**Directory structure:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)